pkg/virt-config/featuregate/active.go

```diff
@@ -0,0 +1,123 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package featuregate
+
+const (
+	ExpandDisksGate       = "ExpandDisks"
+	CPUManager            = "CPUManager"
+	NUMAFeatureGate       = "NUMA"
+	IgnitionGate          = "ExperimentalIgnitionSupport"
+	HypervStrictCheckGate = "HypervStrictCheck"
+	SidecarGate           = "Sidecar"
+	GPUGate               = "GPU"
+	HostDevicesGate       = "HostDevices"
+	SnapshotGate          = "Snapshot"
+	VMExportGate          = "VMExport"
+	HotplugVolumesGate    = "HotplugVolumes"
+	HostDiskGate          = "HostDisk"
+	VirtIOFSGate          = "ExperimentalVirtiofsSupport"
+
+	DownwardMetricsFeatureGate = "DownwardMetrics"
+	Root                       = "Root"
+	ClusterProfiler            = "ClusterProfiler"
+	WorkloadEncryptionSEV      = "WorkloadEncryptionSEV"
+	// DockerSELinuxMCSWorkaround sets the SELinux level of all the non-compute virt-launcher containers to "s0".
+	DockerSELinuxMCSWorkaround = "DockerSELinuxMCSWorkaround"
+	VSOCKGate                  = "VSOCK"
+	// DisableCustomSELinuxPolicy disables the installation of the custom SELinux policy for virt-launcher
+	DisableCustomSELinuxPolicy = "DisableCustomSELinuxPolicy"
+	// KubevirtSeccompProfile indicate that Kubevirt will install its custom profile and
+	// user can tell Kubevirt to use it
+	KubevirtSeccompProfile = "KubevirtSeccompProfile"
+	// DisableMediatedDevicesHandling disables the handling of mediated
+	// devices, its creation and deletion
+	DisableMediatedDevicesHandling = "DisableMDEVConfiguration"
+	// HotplugNetworkIfacesGate enables the virtio network interface hotplug feature
+	HotplugNetworkIfacesGate = "HotplugNICs"
+	// PersistentReservation enables the use of the SCSI persistent reservation with the pr-helper daemon
+	PersistentReservation = "PersistentReservation"
+	// VMPersistentState enables persisting backend state files of VMs, such as the contents of the vTPM
+	VMPersistentState = "VMPersistentState"
+	MultiArchitecture = "MultiArchitecture"
+	// VMLiveUpdateFeaturesGate allows updating certain VM fields, such as CPU sockets to enable hot-plug functionality.
+	VMLiveUpdateFeaturesGate = "VMLiveUpdateFeatures"
+	// When BochsDisplayForEFIGuests is enabled, EFI guests will be started with Bochs display instead of VGA
+	BochsDisplayForEFIGuests = "BochsDisplayForEFIGuests"
+	// NetworkBindingPlugingsGate enables using a plugin to bind the pod and the VM network
+	NetworkBindingPlugingsGate = "NetworkBindingPlugins"
+	// AutoResourceLimitsGate enables automatic setting of vmi limits if there is a ResourceQuota with limits associated with the vmi namespace.
+	AutoResourceLimitsGate = "AutoResourceLimitsGate"
+
+	// Owner: @lyarwood
+	// Alpha: v1.1.0
+	//
+	// CommonInstancetypesDeploymentGate enables the deployment of common-instancetypes by virt-operator
```

Comment:
This looks like a bad rebase, we made this FG Beta during 1.3 and backported the change to 1.2. I wonder if there are other changes lost as a result?